static/js/admin.js

```diff
@@ -163,7 +206,49 @@ exports.setup_page = function () {
         $("#deactivation_stream_modal .stream_name").text(stream_name);
         $("#deactivation_stream_modal").modal("show");
     });
-
```

Comment:
This is because your editor is automatically adding some spaces on the blank line.  If after hitting enter you hit backspace until your cursor is at the left end of the line, you should be good.

Maybe we should add a little script under tools/ that one can use to automatically delete the trailing whitespace -- seems like it should be a pretty safe operation and we could have the linter suggest the tool when it gives that error.